plugins/techdocs/src/alpha.tsx

```diff
@@ -89,68 +131,37 @@ const TechDocsIndexPage = createPageExtension({
  * @alpha
  */
 const TechDocsReaderPage = createPageExtension({
-  id: 'plugin.techdocs.readerPage',
+  id: 'techdocs:reader-page',
```

Comment:
Probably best to wait for 20574

Reply:
Pull request merged, I reverted this change after rebasing this branch with master 🙌🏻